package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
I do not think this is a relevant change.

Reply:
Have you checked my code changes?

Reply:
Removed